src/components/Button/Button.component.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import {ButtonContainer} from './styled';
+
+
+const Button = () => {
+  return (
+    <ButtonContainer>
+        <span>
+            <img className="image" alt="User Avatar" src="https://media.glassdoor.com/sqll/868055/wizeline-squarelogo-1473976610815.png" />
```

Comment:
It will look cleaner if you use a variable for the image URL and pass it to the src.